pallets/xcmp-queue/src/lib.rs

```diff
@@ -122,7 +130,7 @@ pub mod pallet {
 		///
 		/// Events:
 		/// - `OverweightServiced`: On success.
-		#[pallet::weight(weight_limit.saturating_add(1_000_000))]
+		#[pallet::weight((weight_limit.saturating_add(1_000_000), DispatchClass::Operational,))]
 		pub fn service_overweight(
```

Comment:
I find it weird that this function is left unbenchmarked.

Reply:
Yes. What is the reason for this?

Reply:
That seemed a different call to all the rest. Will add it to the benchmarks in another PR.

Reply:
Should I be doing the benchmark for service_overweight in dmp-queue at the same time?